test/testcafe/framework/page-objects/ChallengeOktaVerifyPushPageObject.js

```diff
@@ -46,4 +49,29 @@ export default class ChallengeOktaVerifyPushPageObject extends ChallengeFactorPa
     return this.form.getElement(FORM_INFOBOX_WARNING);
   }
 
+  subtitleExists() {
+    return this.form.elementExist(SUBTITLE_SELECTOR);
+  }
+
+  async autoChallengeInputExists(selector) {
+    return this.form.elementExist(selector);
+  }
+
+  getAutoChallengeCheckboxLabel(selector) {
+    return this.form.getElement(selector);
+  }
+
+  async clickAutoChallengeCheckbox(selector) {
+    await this.t.click(this.form.getElement(selector));
+  }
+
+  async  formWithClassExist() {
```

Comment:
let's rename this to `isOktaVerifyPushChallengeForm()`, so it is a little more explicit what we are testing